model/src/main/kotlin/licenses/LicenseClassifications.kt

```diff
@@ -69,10 +67,10 @@ data class LicenseClassifications(
         categorizations.associate { it.id to it.categories }
     }
 
-    /** A property allowing convenient access to the names of all categories defined. */
+    /** A property allowing convenient access to the names of all categories defined in alphabetical order. */
```

Comment:
Commit message:

> Using an (immutable) sorted set is not necessary.

I guess you mean "a (mutable) sorted set"? (As `SortedSet`s are mutable.)